package.json

```diff
@@ -173,6 +173,9 @@
   "engines": {
     "node": ">=18.0.0"
   },
+  "overrides": {
```

Comment:
This solution is fine for me but I have just one question - instead of overriding that - did you check if upgrading dependencies' versions which use stylus won't resolve that issue too? It would be better. But if you check and it wasn't working then it's fine for me

Reply:
@AleksanderSklorz  i did, unfortunately @nx/webpack 21 is still using stylus 0.64 so until they put the package back up on npm or unless @nx doesn't change the deps we need to handle like this as this is the easiest approach